test/classes/override/override-vdispatch-def-arg.compopts

```diff
@@ -0,0 +1 @@
+--devel
```

Comment:
I suggest having two lines with .compopts:
```
--devel
--no-devel
```
since we expect them to behave identically.

Reply:
Makes sense